test/binary_parse_test.cpp

```diff
@@ -887,8 +887,8 @@ INSTANTIATE_TEST_CASE_P(
          "component 32"},
         {"%2 = OpFunction %2 !31",
          "Invalid function control operand: 31 has invalid mask component 16"},
-        {"OpLoopMerge %1 %2 !7",
-         "Invalid loop control operand: 7 has invalid mask component 4"},
+        {"OpLoopMerge %1 %2 !1027",
```

Comment:
SPIR-V 1.1 introduced DependencyInfinite with bit value 4, so I bumped up the bad case to something larger than anything in the table in SPIR-V 1.3